### DISCUSSION TITLE
"sms"  definition missing from HTAN.json

### DISCUSSION THREAD

**Discussion by bwalsh** [ncihtan/htan-portal#311] [Q&A]
Hello,

I'm working with the OHSU HTAN team.

The context stanza in wget [HTAN.jsonld](https://raw.githubusercontent.com/ncihtan/schematic/main/data/schema_org_schemas/HTAN.jsonld) seems to be missing the `sms` prefix.

Is there a document that describes their use?

['sms:displayName', 'sms:required', 'sms:requiresComponent', 'sms:requiresDependency', 'sms:validationRules']

https://github.com/ncihtan/htan-portal/blob/f496b6173fc7e2553c02e79fe239fe879e0ac56f/lib/dataSchemaHelpers.ts#L20-L23

**Comment by alliecreason** [comment]
Bump @milen-sage, we would appreciate your feedback here.
